README.md

```diff
@@ -95,6 +95,28 @@ We'll insert the following middlewares into the rails stack:
 2. `Rack::SslEnforcer`: enforces HTTPS.
 3. `Rack::Deflater`: compresses responses from the application, can be disabled with `ROO_ON_RAILS_RACK_DEFLATE` (default: 'YES').
 
+### Sidekiq
+
+Unless `SIDEKIQ_ENABLED` is set to false we'll check for the existance of sidekiq gems. Please ensure the following lines are included in your project Gemfile.
+
+```
+source 'https://rubygems.org' do
```

Comment:
Well, see my next comment — if we make caching gems (`bundle package --all`) the norm, then you can just have `roo_on_rails` depend on `sidekiq-pro`, and not have to bother with these comments.

(and possibly issue a nice warning if it's not available)